Backend/Middleware-APD/src/test/java/middlewareapd/repository/MockJWTRepositoryTest.java

```diff
@@ -0,0 +1,45 @@
+package middlewareapd.repository;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+import middlewareapd.model.JWToken;
+
+public class MockJWTRepositoryTest {
+
+    private MockJWTRepository repository;
+
+    @BeforeEach
+    public void setUp() {
+        // Initialize repository and mock tokens
+        repository = new MockJWTRepository();
+    }
+
+    @Test
+    public void testGetTokenByUuid() {
```

Comment:
think it would be good if can briefly describe what the test is testing for

Reply:
Added javadocs to test files (I rmb for the main forgot abt test :'))